TF migration docs - round 6 (#2045)
saucelabs/sauce-docs: docs/test-fairy/integrations/ms-teams.md

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

TestFairy integrates with Microsoft Teams seamlessly, providing human-readable, real-time notifications for your selected events.

1. Create a new channel, or use an existing one to deliver notifications. In the image below, a new channel named `TestFairy-Notifications` is created.
   <img src={useBaseUrl('/img/test-fairy/integrations/ms-teams/01-create-channel.png')} alt="create channel"/>

1. Add a Connector to the channel from the dropdown list.
   <img src={useBaseUrl('/img/test-fairy/integrations/ms-teams/02-add-connector.png')} alt="add connector"/>

1. Select **Configure** or **Add** the **Incoming Webhook** from the dialog.
   <img src={useBaseUrl('/img/test-fairy/integrations/ms-teams/03-select-incoming-webhook.png')} alt="select incoming webhook"/>

> 1. Give the webhook a name; in this case, the webhook will be named `TestFairy Webhook`. You can also give the webhook an image.

Author: ```suggestion
1. Give the webhook a name; in this case, the webhook is named `TestFairy Webhook`. You can also give the webhook an image.
```
